OakCityV2/DetailViewController.h

```diff
@@ -7,16 +7,16 @@
 //
 
 #import <UIKit/UIKit.h>
-
+#import "Job.h"
 @interface DetailViewController : UIViewController
 
-@property (weak, nonatomic) NSString *ttitle;
-@property (weak, nonatomic) NSString *company;
-@property (weak, nonatomic) NSString *desc;
-@property (weak, nonatomic) NSString *url;
-//@property (strong, nonatomic) UIScrollView *scrollView;
-@property (strong, nonatomic) IBOutlet UIWebView *titleLabel;
-@property (strong, nonatomic) IBOutlet UIWebView *companyLabel;
-@property (strong, nonatomic) IBOutlet UIWebView *urlLabel;
+@property (strong, nonatomic) NSString *ttitle;
+@property (strong, nonatomic) NSString *company;
+@property (strong, nonatomic) NSString *desc;
+@property (strong, nonatomic) NSString *url;
```

Comment:
You've carefully built a data object to hold these individual data items.  There's no reason to keep track of them individually.  Instead of having these four properties, you should have one property that is a Job.